RxSwift/RxSwift/Observables/Observable+Creation.swift

```diff
@@ -56,6 +56,17 @@ public func returnElements<E>(values: E ...) -> Observable<E> {
     }
 }
 
+public func from<E, S where S: SequenceType, S.Generator.Element == E>(secuence: S) -> Observable<E> {
```

Comment:
I think it should be "sequence" instead of "secuence". Not a big problem, but we should fix it.